tests/ci/ci_config.py

```diff
@@ -1421,8 +1424,8 @@ def __hash__(self) -> int:
 CHECK_DESCRIPTIONS = [
     CheckDescription(
         StatusNames.SYNC,
-        "A status of the workflow running in the cloud repository against the sync PR",
-        lambda x: x == StatusNames.SYNC,
+        "If it fails, ask a maintainer for help",
+        lambda x: "Sync" in x,
```

Comment:
No, why? It must be the exact name.

Any example where this `in` is necessary?

Reply:
it's a minor thing, i didn't know you change it too

Reply:
The description is subjective; this one is good, too.

But we shouldn't have other `Sync` statuses, hopefully.